vm/src/vm/vm_ops.rs

```diff
@@ -131,7 +131,12 @@ impl VirtualMachine {
     ///   b.rop(b,a)[*], a.op(a,b), b.rop(b,a)
     ///
     /// [*] only when Py_TYPE(a) != Py_TYPE(b) && Py_TYPE(b) is a subclass of Py_TYPE(a)
-    fn binary_op1(&self, a: &PyObject, b: &PyObject, op_slot: &PyNumberBinaryOpSlot) -> PyResult {
+    pub fn binary_op1(
+        &self,
+        a: &PyObject,
+        b: &PyObject,
+        op_slot: &PyNumberBinaryOpSlot,
+    ) -> PyResult {
```

Comment:
Which format config should we use please? When I run `cargo fmt`, it will just change back to one line as before.